raiden/tests/integration/api/test_pythonapi.py

```diff
@@ -28,7 +27,8 @@ def test_token_addresses(raiden_network, token_addresses):
 
 @pytest.mark.parametrize('number_of_nodes', [2])
 @pytest.mark.parametrize('channels_per_node', [0])
-def test_channel_lifecycle(raiden_network, token_addresses, deposit, transport_protocol):
+def test_raidenapi_channel_lifecycle(raiden_network, token_addresses, deposit, retry_timeout):
```

Comment:
nitpick: The reason why it was named simply `test_channel_lifecycle` before is that since it's inside the `test_pythonapi.py` file it's implied that it does test the API. So there is no need to explicitly place the test in some kind of `raidenapi` namespace.